CdpDotnetNetworkChecker/Utils/TrustStore.cs

```diff
@@ -6,24 +6,22 @@ namespace CdpDotnetNetworkChecker.Utils;
 
 public static class TrustStore
 {
-    private const string ConfigKey = "TrustStore";
-
-    public static void SetupTrustStore(this WebApplicationBuilder builder, Logger logger)
+    public static void SetupTrustStore(Logger logger)
     {
         logger.Information("Loading Certificates into Trust store");
-        var certificates = GetCertificates(builder.Configuration, logger);
+        var certificates = GetCertificates(logger);
         AddCertificates(certificates);
     }
 
-    private static List<string> GetCertificates(IConfiguration configuration, Logger logger)
+    private static List<string> GetCertificates(Logger logger)
     {
-        return configuration.GetSection(ConfigKey).GetChildren()
-            .Where(config => config.Value != null && IsBase64String(config.Value))
-            .Select((config, _) =>
+        return Environment.GetEnvironmentVariables().Cast<KeyValuePair<string, string>>()
```

Comment:
Returns a IDictionary - this is how you have to convert to a LINQ expression unfortunately. The alternative is:

```
Dictionary<string, string> certificates = new Dictionary<string, string>();
foreach (KeyValuePair<string, string> entry in Environment.GetEnvironmentVariables())
{
    if (entry.Key.StartsWith("TrustStore"))
    {
        certificates.Add(entry.Key, entry.Value);
    }
}
```